Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -117,6 +117,8 @@ def name(self):
     def validateInputs(self):
         result = dict()
         message = 'Wrong number of {0} runs: {1}. Provide one or as many as sample runs: {2}.'
+        message_value = 'Wrong number of {0} values: {1}. Provide one or as ' \
```

Comment:
This looks to me the same as the line above.

Reply:
It will show `Wrong number of DeltaQ values` instead of `Wrong number of DeltaQ runs`. That sounded a bit better to me.

Reply:
Ok, I missed that.